### PR TITLE
Renames UI references of binary annotation to tag

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Long time coming.. Now that we don't require v1 data any more we can
clean up some old naming in the UI.

Before:
screen shot 2018-11-26 at 10 32 15 pm

After:
screen shot 2018-11-26 at 10 32 03 pm

cc @openzipkin/ui 
cc @wu-sheng  @drolando for language checking

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
you'll notice I fixed the examples which were not good :P

File: zipkin-ui/static/traces_it_IT.properties [hunk 1]
```diff
@@ -22,7 +22,7 @@ modal.title3 = 1. Ricerca per nome dello span
 modal.p4 = Span sono generalmente metodi thrift o endpoints Rails. Questo ti permette di recuperare tracce che accedono ad una particolare parte del servizio.
 modal.title4 = 2. Ricerca per annotazione
 modal.p5 = Il suo valore e' di tipo stringa ed e' quello che puoi utilizzare come parametro di ricerca. Possono essere cose come.
-modal.title5 = 3. Ricerca per chiave/valore
-modal.p6 = Annotazioni di tipo chiave / valore sono un metadato extra attaccato ad un traccia. Esempio: l'url chiamato, il codice di risposta o un'eccezione. Guarda sotto per alcuni esempio.
+modal.title5 = 3. Ricerca per etichetta
+modal.p6 = Etichette sono un metadato extra attaccato ad un traccia. Esempio: l'url chiamato, il codice di risposta o un'eccezione. Guarda sotto per alcuni esempio.
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]:
```suggestion
modal.p6 = Le etichette sono un metadato extra attaccato ad un traccia. Esempio: l'url chiamato, il codice di risposta o un'eccezione. Guarda sotto per alcuni esempio.
```

**Review by drolando — APPROVED** [review]
Fix & ship

**Comment by codefromthecrypt** [comment]
gracie!

**Pull request merged**